.github/workflows/build-wheels.yml

```diff
@@ -0,0 +1,76 @@
+name: Build Wheels
+
+on: 
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+  pull_request:
+    branches-ignore:
+      - docs
+    paths-ignore:
+      - 'docs/**'
+
+jobs:
+  build_wheels:
+    name: Build wheels on ${{ matrix.config.os }} - ${{matrix.python-build}}
+    runs-on: ${{ matrix.config.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        config: 
+          - os: ubuntu-20.04
+            shell: bash
+          # - os: macos-10.15
+          #   shell: bash
+          - os: windows-2019
+            preamble: call "C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Enterprise\\VC\\Auxiliary\\Build\\vcvarsall.bat" x64
+            shell: cmd
+          # These are the platform build strings provided to
+          # cibuildwheel, with wildcarding. See
+          # https://cibuildwheel.readthedocs.io/en/stable/options/#build-skip
+        python-build: ['cp39*64', 'cp310*64']
+    defaults:
+      run:
+        # Annoyingly required here since `matrix` isn't available in the
+        # `shell` property of individual steps.
+        shell: ${{ matrix.config.shell }}
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Bootstrap
+        uses: ./.github/bootstrap_platform
+        env:
+          # As we're building wheels inside a managed environment,
+          # (cibuildwheels) there is no need to install cpython. 
+          OPENASSETIO_CONAN_SKIP_CPYTHON: "True"
+
+      - name: Build wheels
+        run: |
+          ${{ matrix.config.preamble }}
+          pip install cibuildwheel
```

Comment:
I would recommend to pin the version of cibuildwheel. You can use the GitHub action to make it easier to upgrade (with the help of dependabot).

Reply:
Unfortunately because of our reliance on `vcvarsall` (due to us using the Ninja generator, it seems), we can't use the Github action. Something we'll to look into in #669.

We're going to look into our version pinning strategy more broadly in #667.

Otherwise I'm happy for `cibuildwheel` version to be pinned, there are arguments either way.

Reply:
Yeah good idea. Have pinned it to 2.11.1 just like's been done in OTIO.

We used the direct install method for the convenience of being able to run `vcvarsall` on windows platforms, which is what that `config.preamble` is doing above, and have all that config bleed into the environment. For the moment we'll leave that as it stands, and forgo dependabots help in this one case.

Reply:
> there are arguments either way

We've been tending to favour pinning over not pinning when the decision has came up, i've noticed, so will continue on with pinning this particular version, with an eye to conforming it to whatever strategy is decided after the fact.

Reply:
I think for this, as we only find our during a release, I'd def want to opt for pinning - as a failure here is far more of a pain than a PR CI job.